webserver/static/gulpfile.js

```diff
@@ -135,12 +135,17 @@ gulp.task('watch', ['styles', 'scripts'], function () {
     }
 
     if (rebuild) {
-      writeScript(b, resourceName).done(writeManifest);
+      process.stdout.write(`Rebuilding ${resourceName} (${file.event}: ${file.path}) ... `);
+      writeScript(b, resourceName).done(function () {
+        writeManifest();
+        process.stdout.write('done.\n');
+      });
     }
   }
 
+  let watch = require('gulp-watch');
   watch(path.resolve(SCRIPTS_DIR, '**/*.js'), function (file) {
-    _.each(CACHED_BUNDLES, function (bundle, resourceName) {
+    CACHED_BUNDLES.forEach(function (bundle, resourceName) {
```

Comment:
this wasn't working for me, but when I changed from `_` to `.forEach` it did. No idea what's going on